modules/hostap/src/supp_api.c

```diff
@@ -1372,7 +1398,27 @@ int hapd_config_network(struct hostapd_iface *iface,
 			if (!hostapd_cli_cmd_v("set sae_pwe 2")) {
 				goto out;
 			}
-			iface->bss[0]->conf->sae_pwe = 2;
+		} else if (params->security == WIFI_SECURITY_TYPE_WPA_AUTO_PERSONAL) {
+			if (!hostapd_cli_cmd_v("set wpa 2")) {
+				goto out;
+			}
+			if (!hostapd_cli_cmd_v("set wpa_key_mgmt WPA-PSK SAE")) {
```

Comment:
Please put empty line after `if {}` block here and below

Reply:
Yes, updated